src/components/Display.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+const Display = () => {
+    return (
+        <Link to="/monster">
+            <div>display list of monsters</div>
+        </Link>
```

Comment:
This will have to be refactored to allow for the correct monster to be rendered.